web/src/app.js

```diff
@@ -176,10 +193,12 @@ konnektApp.factory('UserService', ['HttpService', '$window', function (HttpServi
           // ha két jelszó nem egyezik hibaüzenet, foglalt névnél hibaüzenet, (hosszra ellenőrizni)
           console.log('registration error 403:', errorResponse);
           logoutUser();
-          $window.location.href = '#!/register';
-          userData.errormessage = errorResponse.data.errors[0].name + ' : ' + errorResponse.data.errors[0].message;
+          // userData.errormessage = errorResponse.data.errors[0].name + ' : ' + errorResponse.data.errors[0].message;
```

Comment:
why the commented out code? are you sure you don't need that error message here?